packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -213,7 +215,7 @@ export class SwapQuoter {
         makerAssetData: string,
         takerAssetData: string,
         takerAssetSellAmount: BigNumber,
-        options: Partial<SwapQuoteRequestOpts> = {},
+        options: Partial<Omit<SwapQuoteRequestOpts, 'quoteReporter'>> = {},
```

Comment:
Yes -- you are correct.  I experimented with moving `quoteReporter` out of `SwapQuoteRequestOpts ` but because of how many level deeps it needs to go, it ended up feeling a bit too heavy + hairy